cmd/opampsupervisor/supervisor/commander/commander.go

```diff
@@ -91,6 +120,14 @@ func (c *Commander) Restart(ctx context.Context) error {
 	return c.Start(ctx)
 }
 
+func envVarMapToEnvMapSlice(m map[string]string) []string {
+	var result []string
+	for key, value := range m {
+		result = append(result, key+"="+value)
+	}
+	return result
+}
```

Comment:
Currently, if the user configures any environment variables then those are the only ones the collector will have when the process begins ([cmd.Env](https://pkg.go.dev/os/exec#Cmd.Env)). I ran into an issue where the collector didn't have access to a standard macOS library because it wasn't found in the collector processes' environment. 

I think we can save the user some headache if we include the environment the supervisor is running with as well when configuring custom env values. Thoughts?

```suggestion
func envVarMapToEnvMapSlice(m map[string]string) []string {
        // let the command initialize the env itself
	if m == nil {
		return nil
	}
	result := os.Environ()
	for key, value := range m {
		result = append(result, key+"="+value)
	}
	return result
}
```